content/feature/example.md

```diff
@@ -5,12 +5,22 @@ Date: "2015-01-17"
 
 
 ### Description
-My fancy description. Explain what your code does, and what is the purpose of it.
+The code changes the Style properties of the Block and configures it such that Scrollbars will be shown only when overflow is present.
 ### Code
-{{< highlight sql  >}}
+<style>
```

Comment:
this will only work for the example.md file, not other gallery items submitted, we want to generalize this fix for all the gallery items.